Sources/HaCWebsiteLib/ViewModels/Hackathons/Hackathon.swift

```diff
@@ -0,0 +1,5 @@
+import HaCTML
+
+public protocol Hackathon: Nodeable {
```

Comment:
Because in the future we will have multiple hackathons. TBH I created this when I wasn't sure if there was going to be a set format or not. Shuold I remove?

Reply:
This would maybe be useful if there was any specific data that we needed `Hackathon`s to have or thought that we would require Hackathons to do something or expose some data in the future. If you can see a use case then leave it in but otherwise let's ditch

Reply:
Yeah, I think this will become useful once we have properly defined event streams, so I will leave it in.